OasisDEX/oasis-borrow
# Fixed copy and stop loss summary when opening vault

# [Fixed copy and stop loss summary when opening vault](https://app.shortcut.com/oazo-apps/story/6855/copy-and-sl-summary-fix-when-opening-vault)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- fixed copy issue in progress bar
- fixed issue when stop loss summary was visible in first step of vault creation
  
## How to test 🧪
  <Please explain how to test your changes>

- check screens from story, `token` in progress bar should be replaced by correct token value
- when opening borrow / multiply vault stop loss summary shouldn't be displayed unless user decide to progress with selected stop loss level and close to option
